include/swift/AST/DiagnosticsSema.def

```diff
@@ -5351,6 +5353,10 @@ ERROR(override_nsobject_hashvalue_error,none,
       "'NSObject.hashValue' is not overridable; "
       "did you mean to override 'NSObject.hash'?", ())
 
+ERROR(override_nsobject_hash_error,none,
+      "`NSObject.hash(into:)` is not overridable; "
+      "subclasses can customize hashing by overriding the `hash` property", ())
```

Comment:
Is there a reason that this message is worded differently from the one for overriding `hashValue`? Typically, diagnostic messages use a concise style, and I think the question posed above is appropriate here ("did you mean to override 'NSObject.hash'?").